py_mmd_tools/mmd_elements.yaml

```diff
@@ -532,3 +750,16 @@ dataset_citation:
         comment: E.g., a dataset landing page
         description: A URL that gives the location of more complete metadata, such as a dataset landing page. A persistent URL is recommended for this attribute. At MET Norway, landing page urls are https://data.met.no/dataset/, plus the dataset UUID. The nc-to-mmd conversion tool automatically creates the dataset citation url, so the metadata_link is only needed if the information is wanted in the NetCDF file.
   other:
+
+# wigos_id is not in MMD but recommended for the netcdf file
+not_mmd.wigos_id:
```

Comment:
I don't really understand why this one is here. I would use the platform/instrument attribute for this...

Reply:
I only included it because it is on https://adc.met.no/node/4 and this is where the current adc_elements.yaml gets it from. But can be removed if it seems unecessary.

Reply:
yes, this should be removed